core/src/main/java/org/radargun/stats/RequestSet.java

```diff
@@ -0,0 +1,61 @@
+package org.radargun.stats;
+
+import org.radargun.Operation;
+
+/**
+ * @author Radim Vansa &lt;rvansa@redhat.com&gt;
+ */
```

Comment:
Some short documentation could be useful - it appears it's only used for transactions, is that right?